driver/cmd/ibm-spectrum-scale-csi/main.go

```diff
@@ -143,9 +146,13 @@ func setContext() context.Context {
 	return ctx
 }
 
-func getLogEnv() string {
+func getLogEnv() (string, string) {
 	level := os.Getenv(logLevel)
-	return level
+	persistentLogEnabled := os.Getenv(persistentLog)
+	if persistentLogEnabled == "" {
```

Comment:
see if something like this help cover the corner cases and case sensitivity cases
if ToUpper(persistentLogEnabled) != "ENABLED" {
 		persistentLogEnabled = "DISABLED"
 	}
 	return level, ToUpper(persistentLogEnabled)

Reply:
If we are going to handle these cases as part of this PR then similar changes can be done for log level env var too

Reply:
Done for both persistent log and log level